packages/near-sdk-js/src/near-bindgen.ts

```diff
@@ -166,29 +166,31 @@ export function NearBindgen({
       }
 
       static _getState(): unknown | null {
-        const rawState = near.storageRead("STATE");
+        const rawState = near.storageRead(bytes("STATE"));
         return rawState ? this._deserialize(rawState) : null;
       }
 
       static _saveToStorage(objectToSave: unknown): void {
-        near.storageWrite("STATE", this._serialize(objectToSave));
+        near.storageWrite(bytes("STATE"), this._serialize(objectToSave));
```

Comment:
Same comment again. Can be unintuitive for devs. What will happen if we omit `bytes()` call?

Reply:
In ts, compile error. In js, runtime error, smart contract panicked, expect Uint8Array for key.